docs/docs/api-reference.md

```
@@ -0,0 +1,54 @@
+| Parameter | Type | Default | Description |
+| :--- | :--- | :--- | :--- |
+| `sigma_spatial` | `float` | `3` | Bilateral spatial sigma |
+| `sigma_range` | `float` | `50` | Bilateral range sigma |
+| `num_colors` / `k` | `int` | `16` | Number of clusters |
+| `max_iter` | `int` | `100` | K-means iterations |
+| `min_area` | `int` | `100` | Minimum contour area |
+| `color_space` | `int` | `0` | `0` = CIE LAB, `1` = sRGB |
+
+## Pipeline Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)